docs/src/index.md

```diff
@@ -46,6 +46,30 @@ interpolate1d!(ftargetcols, xsourcecols, xtargetcols, fsourcecols, Linear(), Fla
 
 The above examples can be run on NVIDIA GPUs by setting `DA = CuArray`.
 
+1D interpolation can also be invoked using a broadcasting call.
+
+```julia
+import ClimaInterpolations.Interpolation1D:
+    Linear, Interpolate1D, interpolate1d!, Flat
+
+FT, DA = Float32, Array
+xmin, xmax, nsource, ntarget = FT(0), FT(2π), 150, 200
+xmintarg, xmaxtarg = xmin, xmax
```

Comment:
let's rename the `xmintarg`, `xmaxtarg` -> `xmintarget`, `xmaxtarget` : I don't think the shorthand notation saves us much, and maintaing `source` and `target` as the terminology in examples makes string searches simpler.